features/content_crop.feature

```diff
@@ -270,7 +269,41 @@ Feature: Cropping the Image Articles
         }
       }
       """
-
+      When we get "/archive/bike"
+      Then we get OK response
+      And we get no "poi"
+      When we patch "/archive/feature_image"
+      """
+      {
+        "associations": {
+          "featureimage": {
+            "_id": "bike",
+            "poi": {"x": 0.1, "y": 0.1},
+            "renditions": {
+              "16-9": {"CropLeft":0,"CropRight":1280,"CropTop":0,"CropBottom":720}
```

Comment:
You could do that but what's the gain? You would have to do 2 separate requests to update an item. You would need to do extra processing to send only the relevant renditions fields to the renditions endpoint.